latest/root/usr/share/container-scripts/postgresql/README.md

```diff
@@ -235,6 +235,54 @@ enough space for the copy;  upgrade failure because of not enough space might
 lead to data loss.
 
 
+Extending image
+----------------
+
+This image can be extended using
+[source-to-image](https://github.com/openshift/source-to-image).
+
+For example to build customized image `new-postgresql`
+with configuration in `~/image-configuration/` run:
+
+
+```
+$ s2i build --assemble-user root ~/image-configuration/ postgresql new-postgresql
```

Comment:
Is `--asemble-user root` realistically useable in openshift?  Shouldn't we rather avoid root user at all?

Reply:
User root is in this solution is used only for building the image using s2i. When the built container is run, it is under postgre user who is only in group postgre.

Reply:
So is this really commonly enabled feature in OpenShift, so we can make it a "general" requirement?  It seems to be pretty dangerous to get root access, even during the build..

Reply:
Other possibility is to add postgres user to root group (as is done e.g in mysql https://github.com/sclorg/mysql-container/blob/master/root-common/usr/libexec/container-setup#L58) but I can't help, it doesn't feel right. I think this issue deserves more discussion. @praiskup @hhorak what do you think?

Reply:
To better understand, what exactly is the 'root' needed for?

Reply:
Actually, that is really good question! The problem with missing write permission (that caused failed s2i builds) on /opt/app-root/src is now managed in Dockerfiles with https://github.com/sclorg/postgresql-container/pull/208/files#diff-4040b6e79b7069db62e62fee9e5c3532R72. fix-permissions might fail, but the build finishes successfully.  Thank you for making this comment and standing your ground! :)